go/integration_test/interpreter/vm_test.go

```diff
@@ -117,6 +117,12 @@ func BenchmarkEmpty(b *testing.B) {
 	}
 }
 
+func BenchmarkStaticOverhead(b *testing.B) {
+	b.Run("1", func(b *testing.B) {
```

Comment:
this line is superfluous.
this line is used in other benchmarks to change some parameter value and name the benchmark after the value. There are no parameters in this  version